CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### Bug fixes
 
+- [#199](https://github.com/babylonlabs-io/babylon-sdk/pull/199) dep: avoid using babylon utilities in demo/ and x/
```

Comment:
Should it be under improvements?

Reply:
I'd say this is a bug. If `x/babylon` depends on Babylon node then 1) the Cosmos BSN will indirectly import a lot of things  that are not necessary, and 2) only Cosmos BSN matching Babylon node's dependencies can use `x/babylon`.

Reply:
Yes I'm also thinking about this and totally agree, https://github.com/babylonlabs-io/babylon-sdk/issues/200